### DISCUSSION TITLE
RealUnit (REALU)

### DISCUSSION THREAD

**Discussion by luziusmeisser** [Frankencoin-ZCHF/Frankencoin#14] [Acceptable Collaterals]
(Placeholder for [REALU](https://realunit.ch/realunit-kaufen/#token) discussion.)

**Comment by AndxyO** [comment]
I think it's ok as a collateral.
When I saw it the first time on aktionariat, I was very skeptical. Perhaps it was just because of the website design. 
I checked the reports and I think they have a good conservative Swiss portfolio (plus some BTC/ETH and good amount of precious Metalls), but you never know where volatility goes in these times.
One good thing is, they are listet on BX and in Germany, so there are earning reports.

**Comment by AndxyO** [comment]
Last WE the REALU Token had a "depeg" (for some time) on the ZCHF platform. There was no real price drawdown with the assets. 
What kind of Oracle is used to get the Price for this asset? The Frankencoin api Bot says Coingecko, but there is no Coingecko Price for REALU.

<img width="338" height="202" alt="benannt" src="https://github.com/user-attachments/assets/b7684a21-3f59-43f2-bfda-bb63a0193afc" />

**Comment by luziusmeisser** [comment]
First price source is a subgraph that sources Aktionariat trading data. But the API key our frontend is using did run out of credits, so the frontend went to coingecko as a fall-back, which is outdated. I think reviving the coingecko price feed is a pending item with Aktionariat.

**Comment by AndxyO** [comment]
Ok, running out of credits shouldn't happen. Not a big deal if the backup is working.

I think Coingecko is not good for a price oracle with low liq assets. 

Coingecko is retrieving prices from different DEXes and CEXes (the ones with most Volume/Liq). If that changes, it takes the Price from the next exchange and if it changes again it takes the next price. So, it can happen on Coingecko that the price of an asset (low DEX/CEX Vol.) can jump up and down the whole time without making any real trading movement.

Sometimes Coingecko does even label an asset as "inactive" when there is no Volume on the retrieved exchanges, even if this asset is still traded on some platforms which are not reterieved.
